test/e2e/page-objects/pages/test-dapp-individual-request.ts

```diff
@@ -0,0 +1,26 @@
+import { Driver } from '../../webdriver/driver';
+
+class TestDappIndividualRequest {
+  private readonly driver: Driver;
+
+  private readonly result = 'main';
+
+  constructor(driver: Driver) {
+    this.driver = driver;
+  }
+
+  /**
+   * Verify the result of the request.
+   *
+   * @param expectedResult - The expected result from the individual request.
+   */
+  async check_expectedResult(expectedResult: string) {
```

Comment:
To be case consistent, maybe `checkExpectedResult`?

Reply:
thank you for the comment, Joao! I agree `checkExpectedResult` makes sense. Changed in the next commit